src/pages/commits/ActiveCommits/BuildVariantCard.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Body } from "@leafygreen-ui/typography";
+
+const { gray } = uiColors;
+
+interface Props {
+  buildVariantDisplayName: string;
+}
+export const BuildVariantCard: React.FC<Props> = ({
+  buildVariantDisplayName,
+}) => <Card key={buildVariantDisplayName}>{buildVariantDisplayName}</Card>;
```

Comment:
key isn't needed here since this is not rendering a list. It would be passed into the component when its rendered in the list.